CHANGELOG.rst

```diff
@@ -7,6 +7,9 @@ Release Notes
 .. 0.7.3 (unreleased)
    ==================
 
+- Added (temporarily) a module to allow multivariate output fits.
+  (multivariate.py) [#137]
```

Comment:
No need to mention something temporary in the change log. Let's keep this under the radar.